ztp/policygenerator/testData/TestNamespace/templates/TestNamespace.yaml

```diff
@@ -12,5 +12,3 @@ spec:
   # Create operators policies that will be installed in all clusters
     - fileName: TestNamespace.yaml
       policyName: "test-sub-ns-policy"
-    - fileName: TestNamespace.yaml
-      policyName: ""
```

Comment:
I think the future case of "PGT generates everything, then SiteConfig injects some of the non-wrapped CRs at day-0" is a good example of a use case where we want both wrapped and non-wrapped generated at the same time.

Reply:
@lack why would we need both wrapped and non-wrapped CRs in this case? the CRs that we generate using PGT for the extra manifests in SiteConfig would be wrapped ones right?

Reply:
We would need the wrapped CRs to apply to the hub cluster, but we also would need the unwrapped CRs because that's what SiteConfig needs in it's extra manifests directory to pass through.

Reply:
ohh ok I understand so then does it make sense to keep the options as either we generate wrapped CRs or we generate both? instead of one or the other? thoughts @imiller0 @serngawy?

Reply:
sounds like u guys miss understand each other; we can generate the CRs only and also we can generate the CRs-wrapped in ACM policy but we cannot do both at the same execution. So when I run $ kustomize build I got the CRs alone OR the CRs wrapped in ACM policy not both in the same execution.

Reply:
added the test back as per discussions